src/inspector/models/abstractinspectormodel.cpp

```diff
@@ -371,6 +371,7 @@ void AbstractInspectorModel::setModelType(InspectorModelType modelType)
 void AbstractInspectorModel::onPropertyValueChanged(const mu::engraving::Pid pid, const QVariant& newValue)
 {
     setPropertyValue(m_elementList, pid, newValue);
+    updateProperties();
```

Comment:
Isn't `loadProperties` enough? (`updateProperties` additionally re-collects the selected elements, which doesn't seem necessary to me, is it?)